src/base/ddl_log.c

```diff
@@ -622,7 +622,7 @@ logddl_make_copy_filename (T_APP_NAME app_name, const char *file_full_path, char
   const char *name_tmp = NULL;
   int retval = 0;
 
-  if (file_full_path == NULL || copy_filename == NULL || buf_size < 0)
+  if (file_full_path == NULL || copy_filename == NULL)
```

Comment:
I think it would be better to remove the buf_size argument and specify PATH_MAX directly.

Reply:
I think the buf_size parameter should be kept. The reason is that the size of the string can change.